app/middleware/getPharmacies.js

```diff
@@ -3,14 +3,9 @@ const log = require('../lib/logger');
 
 function getPharmacies(req, res, next) {
   const searchPoint = res.locals.coordinates;
-  // TODO: Implement this
-  // const limits = { nearby: 10, open: 3 };
-
-  const latitude = searchPoint.latitude;
-  const longitude = searchPoint.longitude;
 
   const baseUrl = process.env.API_BASE_URL;
-  const url = `${baseUrl}/nearby?latitude=${latitude}&longitude=${longitude}`;
+  const url = `${baseUrl}/nearby?latitude=${searchPoint.latitude}&longitude=${searchPoint.longitude}&limits:results:open=1&limits:results:nearby=3`;
```

Comment:
Useful to have a query parameter naming convention to group related parameters - why ':' rather than, say, '.' ?

Just checked and node doesn't just handle '.' (i.e. res.query.limits.results.open doesn't work need to use res.query['limits.results.open']). That would have been just too elegant :)

Should latitude and longitude have a similar convention (searchPoint:latitude)?